components/eamxx/src/control/atmosphere_surface_coupling_exporter.cpp

```diff
@@ -196,6 +202,42 @@ void SurfaceCouplingExporter::initialize_impl (const RunType /* run_type */)
   // Copy data to device for use in do_export()
   Kokkos::deep_copy(m_column_info_d, m_column_info_h);
 
+  // Set the number of exports from eamxx or set to a constant, default type = EAMXX
+  using vos_type = std::vector<std::string>;
+  using vor_type = std::vector<Real>;
+  m_export_source     = view_1d<DefaultDevice,ExportType>("",m_num_scream_exports);
+  auto export_source_h = Kokkos::create_mirror_view(m_export_source);
+  Kokkos::deep_copy(m_export_source,EAMXX);  // The default is that all export variables will be derived from the EAMxx state.
+  m_num_eamxx_exports = m_num_scream_exports;
```

Comment:
I think this is confusing, especially in view of EAMxx in E3SMv4 (where there will be no "scream"). But even for scream, it might be confusing ("what's the difference between eamxx and scream?" on may ask).

I think the `eamxx` part was meant to reflect the enum value `EAMXX`. Perhaps that's a bit ambiguous? Maybe we could change `eamxx/EAMXX` with `computed/COMPUTED`?

Reply:
That's a fair point, I can change to `COMPUTED`, or `FROM_MODEL` may also be descriptive, since `computed` is also a bit ambiguous.  The prescribed ones are computed from somewhere too, and not all fluxes are computed in surface coupling, some are just taken from model state.

Reply:
Can we change `m_num_eamxx_exports` too then? I see this line becoming puzzling when scream becomes eamxx...